src/LCT.Common/Model/Config.cs

```diff
@@ -22,6 +22,7 @@ public class Config
         public string[] JfrogNugetRepoList { get; set; }
         public string[] JfrogMavenRepoList { get; set; }
         public string[] JfrogPythonRepoList { get; set; }
+        public string[] JfrogConanRepoList { get; set; }
         public string[] DevDependentScopeList { get; set; }
```

Comment:
Are there changes in appSettings.json file?If yes then push those as well

Reply:
Yes, have updated the appSettings.json with the Conan configurations now.